src/components/app.rs

```diff
@@ -19,7 +19,6 @@ extern "C" {
 pub fn App() -> impl IntoView {
     let (program_str, set_program_str) = create_signal("".to_string());
     let (run_result, set_run_result) = create_signal::<Option<Result<String, String>>>(None);
-    let (is_running, set_is_running) = create_signal(false);
```

Comment:
I'm not sure why you are removing this code. `is_running` disables the run button while the program is running. I don't see the button as gray while the program is running, so this behavior might not work as intended. In this case, we should fix `is_running` rather than removing it.

I reverted the Rust changes of your commit and pushed them on master.